core/components/form-fields/shared/parse-fields.ts

```diff
@@ -1,4 +1,4 @@
-import { UpdateCustomerAddressInput } from '~/app/[locale]/(default)/account/(tabs)/addresses/_actions/update-address';
+import { UpdateCustomerAddressInput } from '~/app/[locale]/(default)/account/(tabs)/addresses/edit/[slug]/_actions/update-address';
```

Comment:
Trying to figure out how to get input values without relying on a specific mutation. 🤔

Reply:
Could you create a fragment to share across these mutations? Then use the fragment for types?

Reply:
Created a fake mutation, other solution was to type it myself.